app/uk/gov/hmrc/gform/views/summary/TextFormatter.scala

```diff
@@ -67,7 +65,7 @@ object TextFormatter {
   ): String =
     (textConstraint, prefix, suffix) match {
       // format: off
-      case (IsPositiveNumberOrNumber(PositiveNumberOrNumber(_, maxFractionalDigits, roundingMode, unit)), p, s)  =>   prependPrefix(p) + formatNumber(currentValue, maxFractionalDigits, roundingMode,  s.map(_.localised).orElse(unit))
+      case (IsPositiveNumberOrNumber((_, maxFractionalDigits, roundingMode, unit)), p, s)  =>   prependPrefix(p) + formatNumber(currentValue, maxFractionalDigits, roundingMode,  s.map(_.localised).orElse(unit))
       case (_: Sterling, _, _)                                                                                   =>   formatSterling(currentValue)
       case (_, p, s)                                                                                             =>   prependPrefix(p) + currentValue + appendSuffix(s)
       case _                                                                                                     =>   currentValue
```

Comment:
Either get rid of `// format: off` or keep an alignment of `=>` in this block of code

And why is there triple space after `=>`?

And `(...)` can be dropped from `IsPositiveNumberOrNumber` as before